io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -61,22 +62,26 @@ def setUp(self):
         elif self.raid_level == 'r50' or self.raid_level == 'r60':
             self.pdperarray = 3
         if self.raid_level == 'r0':
-            if 'cc' in str(self.name):
-                self.skip("CC not applicable for Raid0")
+            if 'cc' in str(self.name) or 'offline' in str(self.name):
+                self.skip("Test not applicable for Raid0")
+        self.write_policy = ['WT', 'WB', 'AWB']
+        self.read_policy = ['nora', 'ra']
+        self.io_policy = ['direct', 'cached']
+        self.stripe = [64, 128, 256, 512, 1024]
 
     def test_createall(self):
 
         """
         Function to create different raid level
         """
-        write_policy = ['WT', 'WB', 'AWB']
-        read_policy = ['nora', 'ra']
-        io_policy = ['direct', 'cached']
-        strip = [64, 128, 256, 512, 1024]
-        for write in write_policy:
-            for read in read_policy:
-                for iopolicy in io_policy:
-                    for stripe in strip:
+        self.write_policy = ['WT', 'WB', 'AWB']
+        self.read_policy = ['nora', 'ra']
+        self.io_policy = ['direct', 'cached']
+        stripe = [64, 128, 256, 512, 1024]
```

Comment:
My bad. Any other change is required, If not I can quickly do reset and resend the patch.